molecule/default/molecule.yml

```
@@ -0,0 +1,29 @@
+---
+dependency:
+  name: galaxy
+driver:
+  name: podman
+platforms:
+  - name: oc_mariadb_centos_8
+    image: docker.io/pycontribs/centos:8
```

Since CentOS 8 reached its end of life, does it make sense to use CentOS Stream 8 instead?

```suggestion
    image: docker.io/pycontribs/centos:stream8
```

---

See comment above.

---

In the end I changed this to stream 8 and provided a new Dockerfile for this.